client/my-sites/checkout/cart/popover-cart.jsx

```diff
@@ -16,7 +16,6 @@ import { withShoppingCart } from '@automattic/shopping-cart';
  */
 import CartBody from './cart-body';
 import CartBodyLoadingPlaceholder from './cart-body/loading-placeholder';
-import CartMessages from './cart-messages';
```

Comment:
Is `<PopoverCart>` within `<CalypsoShoppingCartProvider>`? It uses `withShoppingCart` and I'm still looking through its position within the component tree.

Reply:
Yup! See https://github.com/Automattic/wp-calypso/pull/47605

Reply:
I see it wrapped as part of `<PlansNavigation>` but I'm having a hard time finding it through `<HeaderCart>` as I'm not really sure how the `<Main>` component is used where it eventually ends up.